Improvements to readme.md (#110)
eisopux/checker-framework-vscode: README.md

## Getting Started

> Please refer to the "Developer's Guide" before getting started.

Reviewer: ```suggestion
You can either install the extension from the
[VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
or install a locally-built version of the extension.
Please refer to the "Developer's Guide" below for instructions on how to build from source.
```

## Developer's Guide

> To set up and build from the command line, perform the following steps:

Reviewer: ```suggestion
To set up your development environment, perform the following steps:
```

> 1. Go to Node.Js (https://nodejs.org/en/) and install the newest Node.js program specific to your operating system (OS).

Reviewer: ```suggestion
1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
```

> 2. Go to Git (https://git-scm.com/downloads) and install a Git program specific to your OS.

Reviewer: ```suggestion
2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).
```

> 3. Open command prompt (CMD) (admin) Install dependencies using `yarn install`.

Reviewer: ```suggestion
3. Install dependencies using `yarn install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.) See the [Yarn homepage](https://yarnpkg.com/).
```

> 4. In CMD (admin) `corepack enable`.

Reviewer: ```suggestion
4. Install the corepack utility by running `corepack enable`, as superuser. See the [Corepack homepage](https://github.com/nodejs/corepack).
```

> 5. CMD (user) create a working directory `mkdir` and change into it `cd`.

Reviewer: ```suggestion

To start working on this project, perform the following steps:
1. Create a working directory for the project and change into it.
```

> 7. In the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git` (adapt to the web URL specific to your branch). 

Reviewer: ```suggestion
2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git` (adapt the URL to your specific fork and branch). 
```

> 8. Change into `cd` `checker-framework-vscode` Run `yarn global add vsce`.

Reviewer: ```suggestion
3. Change into the cloned directory `cd checker-framework-vscode`.
4. Run `yarn global add vsce`.
```

> 9. In the same directory, run `vsce package` to generate a `.vsix` file 
> 10. Run `code --install-extension my-extension-0.0.1.vsix`, replacing `my-extension-0.0.1.vsix` with the created `.vsix` file. 

Reviewer: ```suggestion

To build the extension from the command line:
1. In the `checker-framework-vscode` directory, run `vsce package` to generate a `.vsix` file 
2. Run `code --install-extension my-extension-0.0.1.vsix`, replacing `my-extension-0.0.1.vsix` with the created `.vsix` file. 
```

## Developer's Guide

To set up your development environment, perform the following steps:
1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).

> 3. Install dependencies using `yarn install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.) See the [Yarn homepage](https://yarnpkg.com/).
> 4. Enable the corepack utility by running `corepack enable`, as superuser. See the [Corepack homepage](https://github.com/nodejs/corepack).

Reviewer: ```suggestion
3. Install dependencies using `yarn install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
   See the [Yarn homepage](https://yarnpkg.com/).
4. Enable the corepack utility by running `corepack enable`, as superuser.
   See the [Corepack homepage](https://github.com/nodejs/corepack).
```

To start working on this project, perform the following steps:
1. Create a working directory for the project and change into it.

> 2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git` (adapt the URL to your specific fork and branch). 

Reviewer: ```suggestion
2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
   (adapt the URL to your specific fork and branch). 
```

3. Change into the cloned directory `cd checker-framework-vscode`.

> 4. Run `yarn global add vsce`.

Reviewer: Does this step need to be performed in the project directory? Or should it be moved to the `set up your development environment` above? Can you find a link that explains what this does?

To build the extension from the command line:
1. In the `checker-framework-vscode` directory, run `vsce package` to generate a `.vsix` file 

> 2. Run `code --install-extension my-extension-0.0.1.vsix`, replacing `my-extension-0.0.1.vsix` with the created `.vsix` file. 

Reviewer: Can you adapt the `.vsix` file name to what is actually generated?
